ARGS_ERROR = IndexError(
    "Invalid arguments!\nUsage:\npython3 tdd-monitor.py /path/to/tested/file.py  /path/to/test/file"
)